# Move base node parameters. For full documentation of the parameters in this file, please see
#
#  http://www.ros.org/wiki/move_base
#
shutdown_costmaps: false # Default false

controller_frequency: 20.0 # Default 20.0
controller_patience: 3.0 # Default 15.0 - I just get impatient

planner_frequency: 1.0
planner_patience: 5.0 # Default 5.0

oscillation_timeout: 10.0 # Default 0.0, i.e. never
oscillation_distance: 0.2 # Default 0.5

# For exploration
clearing_rotation_allowed: false
#recovery_behavior_enabled: false # If you do this, at some point the robot will abort EVERY objective!
# My theory is that it never clears the cost map and eventually it just get stuck for good.

# local planner - default is trajectory rollout
base_local_planner: "dwa_local_planner/DWAPlannerROS"
